Infrastructure/gpg-ipdeny/nginx.conf

```diff
@@ -4,7 +4,7 @@ daemon off;
 error_log /dev/stderr;
 
 events {
-  worker_connections 1024;
+  worker_connections 10000;
```

Comment:
did this do anything in the end? if not I'd vote for keeping it as the default 1024

Reply:
Yeah, it fixed my initial errors - now we're seeing timeouts and 500s but before I was seeing connections being dropped when it reached the 1024 threshold and couldn't take on more. 10000 was recommended on a StackOverflow post about configuring these values for live environments, which is why I went for it!